README.md

```diff
@@ -1,75 +1,168 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Create a local directory that you want to clone the repository.
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command git clone https://github.com/codehass/Awesome-books.git
+
+- Go to the repository folder using command prompt cd Awesome-books.
+
+- Install the dev dependencies for linters run npm install.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ### Prerequisites
 
+In order to run this project you need:
+
+-Google chrome
+
+-Internet Explorer
+
+-Any broswer
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ### Setup
 
-### Install
+Clone this repository to your desired folder:
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command git clone https://github.com/codehass/Awesome-books.git
+
+- Go to the repository folder using command prompt cd Awesome-books.
+
+- Install the dev dependencies for linters run npm install.
+
 
-### Usage
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Run tests
+<!-- Author -->
 
-### Deployment
+👤 **Author-01**
 
+- GitHub: [@codehass](https://github.com/codehass)
+- Twitter: [@hassanelourdy](https://twitter.com/hassanelourdy)
+- LinkedIn: [Hassan El OUARDY](https://www.linkedin.com/in/hassan-el-ouardy-360b99169/)
 
+👤 **Author-02**
 
-## Authors
+- GitHub: [@App-Developer8866](https://github.com/App-Developer8866/)
+- Twitter: [@MMuttrafRasool](https://twitter.com/MMuttrafRasool)
+- LinkedIn: [Muhammad Muttraf Rasool](https://www.linkedin.com/in/muhammad-muttraf-rasool-421819202)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+<!-- CONTRIBUTING -->
 
-## 🤝 Contributing
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/codehass/Awesome-books/issues).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
-## 📝 License
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [Optional] License link is broken. Let's kindly fix :wrench: it to render our README file professional :+1: